### PR TITLE
[Next.js][Personalize] Switchover to new CDP hostname for personalize middleware

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Description / Motivation
* Update CDP hostname according to the provided updates

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/nextjs-personalize/.env [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Your Sitecore CDP REST API base URL
-NEXT_PUBLIC_CDP_API_URL=https://api.boxever.com
+NEXT_PUBLIC_CDP_API_URL=https://api-engage-eu.sitecorecloud.io
```
Comment (ambrauer) [inline packages/create-sitecore-jss/src/templates/nextjs-personalize/.env:2]: Should we actually use the dev endpoint? This is non-region specific and actually makes more sense to start with from a dev-perspective.
```suggestion
NEXT_PUBLIC_CDP_API_URL=https://api-engage-dev.sitecorecloud.io
```

File: packages/create-sitecore-jss/src/templates/nextjs-personalize/.env [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 # Your Sitecore CDP client key
 NEXT_PUBLIC_CDP_CLIENT_KEY=
 
 # Your Sitecore CDP target URL
-NEXT_PUBLIC_CDP_TARGET_URL=https://api.boxever.com/v1.2
+NEXT_PUBLIC_CDP_TARGET_URL=https://api-engage-eu.sitecorecloud.io/v1.2
```
Comment (ambrauer) [inline packages/create-sitecore-jss/src/templates/nextjs-personalize/.env:8]:
```suggestion
NEXT_PUBLIC_CDP_TARGET_URL=https://api-engage-dev.sitecorecloud.io/v1.2
```

**Review by ambrauer — CHANGES_REQUESTED** [review]

File: packages/create-sitecore-jss/src/templates/nextjs-personalize/.env (hunk 1 above)
Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/templates/nextjs-personalize/.env]: @ambrauer Good point, replaced

**Review by ambrauer — APPROVED** [review]
👍

**Pull request merged**